pkg/app/mysql.go

```diff
@@ -0,0 +1,227 @@
+package app
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	crv1alpha1 "github.com/kanisterio/kanister/pkg/apis/cr/v1alpha1"
+	"github.com/kanisterio/kanister/pkg/format"
+	"github.com/kanisterio/kanister/pkg/kube"
+	"github.com/kanisterio/kanister/pkg/log"
+	"github.com/kanisterio/kanister/pkg/testutil"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+type chartInfo struct {
+	helmrepoURL   string
+	helmChartName string
+	helmAppname   string
+}
+
+type MysqlDB struct {
+	cli       kubernetes.Interface
+	namespace string
+	password  string
+	chart     chartInfo
+}
+
+func NewMysqlDB(helmrepoURL, helmChart, helmAppname string) App {
+	return &MysqlDB{
+		password: "mysecretpassword",
+		chart: chartInfo{
+			helmrepoURL:   helmrepoURL,
+			helmChartName: helmChart,
+			helmAppname:   helmAppname,
+		},
+	}
+}
+
+func (mdb *MysqlDB) Init(ctx context.Context) error {
+
+	cfg, err := kube.LoadConfig()
+	if err != nil {
+		return nil
+	}
+
+	mdb.cli, err = kubernetes.NewForConfig(cfg)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (mdb *MysqlDB) Install(ctx context.Context, namespace string) error {
+
+	mdb.namespace = namespace
+
+	cli := testutil.NewHelmClient()
+	log.Print("Adding repo.")
+	err := cli.AddRepo(ctx, mdb.chart.helmChartName, mdb.chart.helmrepoURL)
+	if err != nil {
+		log.WithError(err).Print("Failed to add helm repo.")
+	}
+
+	log.Print("MysqlDB: Installing mysql instance using helm.")
+	err = cli.Install(ctx, mdb.chart.helmChartName+"/"+mdb.chart.helmAppname, "mysql-test", mdb.namespace, map[string]string{"mysqlRootPassword": mdb.password, "persistence.enabled": "false", "imagePullPolicy": "Always"})
+	if err != nil {
+
+		log.WithError(err).Print("Failed to install mysql.")
+	}
+
+	return nil
+}
+
+func (mdb *MysqlDB) IsReady(ctx context.Context) (bool, error) {
+	log.Print("Waiting for the mysql instance to be ready.")
+	ctx, cancel := context.WithTimeout(ctx, time.Minute*5)
+	defer cancel()
+	err := kube.WaitOnDeploymentReady(ctx, mdb.cli, mdb.namespace, "mysql-test")
+	if err != nil {
+		return false, err
+	}
+	log.Print("Mysql instance is ready.")
+	return true, nil
+
+}
+
+func (mdb *MysqlDB) Object() crv1alpha1.ObjectReference {
+
+	return crv1alpha1.ObjectReference{
+		Kind:      "deployment",
+		Name:      "mysql-test",
+		Namespace: mdb.namespace,
+	}
+}
+
+func (mdb *MysqlDB) Uninstall(ctx context.Context) error {
+	log.Print("MysqlDB: UnInstalling mysql instance.")
```

Comment:
@viveksinghggits let's use the following format for logging accross all the apps 
`log.Info/Debug/Error().Print("msg", field.M{"app": app.name, additional_fields... })`
Let me know what you think.